install/image-references

```diff
@@ -30,3 +30,27 @@ spec:
     from:
       kind: DockerImage
       name: quay.io/openshift/origin-gcp-machine-controllers:v4.0.0
+  - name: metal3-baremetal-operator
+    from:
+      kind: DockerImage
+      name: quay.io/openshift/origin-baremetal-operator:v4.2.0
+   - name: metal3-ironic
+    from:
+      kind: DockerImage 
+      name: docker://brew-pulp-docker01.web.prod.ext.phx2.redhat.com:8888/openshift/ose-ironic:v4.2.0
```

Comment:
These should all match the pattern used by the others, like quay.io/openshift/origin-ironic:v4.2.0

If it's not there, maybe we need some mirroring configuration added to openshift/release first

Reply:
and remember that these are mostly just placeholders and get replaced with something else when the release image gets built